tests/transformer/test_yield.py

```diff
@@ -34,7 +34,7 @@ def test_yield_from():
     def my_external_generator():
         my_list = [1, 2, 3, 4, 5]
         for elem in my_list:
-            yield elem
+            yield(elem)
```

Comment:
i am not sure if that was done by you or any code formater, but it should stay:

```python
yield elem
```

Reply:
When doing #232 we moved the tests in `src/` folder, so the linters from https://github.com/zopefoundation/RestrictedPython/blob/48165265cba59e7ef9110f2a1646e0d9e706aea3/tox.ini#L59-L60 applied, so we changed the code to pass the lint steps at the same time.

This commit was done automatically with  `git revert` command so it brings us back to the "before" state.

If we want to apply linters to the `tests` folder, that's another story, the linter commands are generated from https://github.com/zopefoundation/meta/blob/b5df3766ff8923477f3d24729b19504f0c401a2e/config/default/tox-lint.j2#L7-L8 and they don't cover `tests` folder. If we want to include lint in `tests` folder it seem better to me to treat this in a separate issue and limit the scope here to just revert.